server/daos/purchasedProducts.js

```diff
@@ -0,0 +1,4 @@
+import db from '@database/models';
+
+export const insertPurchasedProducts = async args =>
+  db.purchasedProducts.create(args).catch(() => 'Error while adding purchased products');
```

Comment:
let's log the db error and append that to the error
Also this will return a string. Not an error. So the calling function won't know that the insert failed
```
export const transformSQLError = e =>
    (e.errors || []).map(err => err.message).join('. ') || e.original;
```